core/src/components/UserMenu/ProfileUserMenuEntry.vue

```diff
@@ -0,0 +1,140 @@
+<!--
+  - @copyright 2023 Christopher Ng <chrng8@gmail.com>
+  -
+  - @author Christopher Ng <chrng8@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+-->
+
+<template>
+	<li :id="id"
+		class="menu-entry">
+		<component :is="profileEnabled ? 'a' : 'span'"
+			class="menu-entry__wrapper"
+			:class="{
+				active,
+				'menu-entry__wrapper--link': profileEnabled,
+			}"
+			:href="profileEnabled ? href : null"
```

Comment:
Works the same by better for type checking

```suggestion
			:href="profileEnabled ? href : undefined"
```